### PR TITLE
Add argument-range check for GunnKinzer1949 interpolation of terminal velocity

### PR DESCRIPTION
**Pull request by abulenok**
closes https://github.com/open-atmos/PySDM/issues/1097

### CODE REVIEW

**Comment by abulenok** [comment]
TODO: decide where to place terminal velocity tests

File: examples/PySDM_examples/Berry_1967/example.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 def main(plot: bool, save):
     with np.errstate(all="ignore"):
-        u_term_approxs = (gunn_and_kinzer.Interpolation,)
+        u_term_approxs = (gunn_and_kinzer.GunnKinzer1949,)
```
Comment (slayoo) [inline examples/PySDM_examples/Berry_1967/example.py:18]: should these now be strings passed to Formulae ctor?

File: PySDM/dynamics/terminal_velocity/gunn_and_kinzer.py [hunk 2]
```diff
@@ -107,7 +107,10 @@ def __init__(self, particulator, small_r_limit=None):
         rbf = Rbf(ir, iu)
         self.factor = 100000
         num = 6 * self.factor // 1000 + 1
-        space, step = np.linspace(0, 0.6 * const.si.cm, num, retstep=True)
+
+        self.minimum_radius = 0
```
Comment (slayoo) [inline PySDM/dynamics/terminal_velocity/gunn_and_kinzer.py:111]: unused?

File: PySDM/backends/impl_thrust_rtc/storage.py [hunk 3]
```diff
@@ -419,6 +424,9 @@ def _to_host(self):
         def amin(self):
             return Impl.amin(self.data)
 
+        def amax(self):
```
Comment (slayoo) [inline PySDM/backends/impl_thrust_rtc/storage.py:427]: perhaps, let's add a minimal unit test for this addition

**Review by slayoo — CHANGES_REQUESTED** [review]

File: examples/PySDM_examples/Berry_1967/example.py (hunk 1 above)
Comment (abulenok) [reply examples/PySDM_examples/Berry_1967/example.py]: applied

File: PySDM/backends/impl_thrust_rtc/storage.py (hunk 3 above)
Comment (abulenok) [reply PySDM/backends/impl_thrust_rtc/storage.py:427]: added

**Comment by abulenok** [comment]
@slayoo WDYT about placing `test_terminal_velocity.py` in `tests/unit_tests/physics`

**Comment by slayoo** [comment]
> @slayoo WDYT about placing `test_terminal_velocity.py` in `tests/unit_tests/physics`

The tested code mostly resides in `dynamics/terminal_velocity`, why not then place it in `tests/unit_tests/dynamics/test_terminal_velocity.py`?

**Comment by slayoo** [comment]
@abulenok, there are three TODO labels in `PySDM/dynamics/terminal_velocity/gunn_and_kinzer.py` which still refer to #348, should we just remove them:
```python
# TODO #348 implement in backend logic
# TODO #348 T, p dependence
# TODO #348 move constants to physics.constants
```

**Review by slayoo — DISMISSED** [review]

**Comment by abulenok** [comment]
> @abulenok, there are three TODO labels in `PySDM/dynamics/terminal_velocity/gunn_and_kinzer.py` which still refer to #348, should we just remove them:
> 
> ```python
> # TODO #348 implement in backend logic
> # TODO #348 T, p dependence
> # TODO #348 move constants to physics.constants
> ```

Maybe a new issue for these? Or even separate smaller issues

**Comment by slayoo** [comment]
> > @abulenok, there are three TODO labels in `PySDM/dynamics/terminal_velocity/gunn_and_kinzer.py` which still refer to #348, should we just remove them:
> > ```python
> > # TODO #348 implement in backend logic
> > # TODO #348 T, p dependence
> > # TODO #348 move constants to physics.constants
> > ```
> 
> Maybe a new issue for these? Or even separate smaller issues

Just renamed #348

**Review by slayoo — APPROVED** [review]

**Pull request merged**